packages/venia-concept/src/components/App/renderRoutes.js

```diff
@@ -3,11 +3,16 @@ import { Switch, Route } from 'react-router-dom';
 import { Page } from '@magento/peregrine';
 import ErrorView from 'src/components/ErrorView/index';
 import CreateAccountPage from 'src/components/CreateAccountPage/index';
+import Search from 'src/RootComponents/Search';
 
 const renderRoutingError = props => <ErrorView {...props} />;
+const search = () => <Search />;
 
 const renderRoutes = () => (
     <Switch>
+        <Route exact path="/search.html">
+            <Page>{search}</Page>
+        </Route>
```

Comment:
If we're handling the route `/search.html` here, why would we render a `Page`? I don't think `Search` should be a RootComponent if it's a client route rather than a server one.

Can we just render a `<SearchPage>` component instead of `<Page>` here?

Reply:
This is still outstanding.

Reply:
Done